src/Sylius/Bundle/CoreBundle/Form/Extension/ChannelTypeExtension.php

```diff
@@ -67,12 +66,6 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->add('taxCalculationStrategy', TaxCalculationStrategyChoiceType::class, [
                 'label' => 'sylius.form.channel.tax_calculation_strategy',
             ])
-            ->add('themeName', ThemeNameChoiceType::class, [
```

Comment:
I think in should be mentioned in UPGRADE guide as somebody already in headless context could have removed this field, which will crash now.